src/index.js

```diff
@@ -0,0 +1,287 @@
+// Wave 2
+let state = {
+  temp: 75,
+};
+
+const images = [
+  'url("./images/image1.jpg")',
+  'url("./images/image2.jpg")',
+  'url("./images/image3.jpg")',
+  'url("./images/image4.jpg")',
+  'url("./images/image5.jpg")',
+  'url("./images/image6.jpg")',
+];
```

Comment:
It would better follow [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns) if you defined classes (like `rain`) and then set these as the background image in CSS.